### ISSUE TITLE
Trials matches not showing up in Activity History

### ISSUE THREAD

**Issue by nev-r** [Bungie-net/api#1371]
not None filter, not Trials filters, etc. :(
see #1362 , #1349

**Comment by jshaffstall-bng** [comment]
Filed as TFS 966381, and we should have this fixed for new activities soon.

Unfortunately, Trials activities from today, but before the fix, won't show up until we do some sort of fixup, which we don't have an ETA for.

**Comment by jshaffstall-bng** [comment]
This should be fixed now for new Trials of Osiris activities going forward.

For example:
https://www.bungie.net/en/PGCR/7595340314?character=2305843009262661548

**Comment by brianbolnick** [comment]
@jshaffstall-bng no need to reopen, just curious where you’ll be posting when this data is back filled? Just looking to stay tuned

**Comment by jshaffstall-bng** [comment]
Because the interested parties are likely subscribed to each issue, we'll try to leave updates on these closed threads.

That said, we won't have an update for you until January, or possibly later.

**Comment by brianbolnick** [comment]
All I needed to know. Thanks!!

**Comment by brianbolnick** [comment]
@jshaffstall-bng assuming this is semi-related so keeping it in this thread. I've noticed the data from the historical stats endpoint isn't being updated. For example, I played a few nights ago on this character but this endpoint is coming up entirely empty:

`https://www.bungie.net/Platform/Destiny2/1/Account/4611686018449620150/Character/2305843009265284017/Stats/?modes=84`

 Not sure if it's just trials or not, but figured I'd give you a heads up in case it is something separate. 

Thanks!
